packages/discord.js/typings/index.d.ts

```diff
@@ -6542,6 +6558,14 @@ export interface PartialMessage
 
 export interface PartialMessageReaction extends Partialize<MessageReaction, 'count'> {}
 
+export interface PartialPoll
+  extends Partialize<
+    Poll,
+    'allowMultiselect' | 'answers' | 'layoutType' | 'expiresTimestamp' | 'question' | 'resultsFinalized' | 'message'
+  > {}
```

Comment:
I believe this matches the current behavior. Type tests will need to be updated

```suggestion
export interface PartialPoll
  extends Partialize<
    Poll,
    'allowMultiselect' | 'layoutType' | 'expiresTimestamp',
    null,
    'question' | 'message'
  > {
    question: { text: null };
    message: PartialMessage;
  }
```